### PR TITLE
Vagrant: Update go version to 1.11

### PR DESCRIPTION
**Pull request by cgbaker**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -137,8 +137,8 @@ def configureLinuxProvisioners(vmCfg)
 		path: './scripts/vagrant-linux-priv-rkt.sh'
 
 	vmCfg.vm.provision "shell",
-		privileged: true,
-		path: './scripts/vagrant-linux-priv-ui.sh'
+		privileged: false,
+		path: './scripts/vagrant-linux-unpriv-ui.sh'
```
Comment (nickethier) [inline Vagrantfile]: You're going to need to actually rename the scripts if you're going to reference it by this new name.

Comment (cgbaker) [reply Vagrantfile]: it's there 😄

Comment (cgbaker) [reply Vagrantfile]: screen shot 2018-10-05 at 3 50 38 pm

Comment (nickethier) [reply Vagrantfile]: Doh!

**Pull request merged**